src/worker.cpp

```diff
@@ -486,11 +486,43 @@ void Worker::setDeviceTreeAndJson()
 
     if (fitConfigVal.find(devTreeFromJson) != std::string::npos)
     { // fitconfig is updated and correct JSON is set.
-
-        if (isSystemVPDOnDBus())
+        try
+        {
+            std::string l_backupRestoreJsonPath = getBackupRestoreJsonPath();
+            m_backupRestoreJson = utils::getParsedJson(l_backupRestoreJsonPath);
```

Comment:
created member variable to avoid loading backup and restore config json in case of calling backupAndRestore() from manager.